Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/validationresult/DccTicketingValidationResultFragment.kt

```diff
@@ -0,0 +1,69 @@
+package de.rki.coronawarnapp.dccticketing.ui.validationresult
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.navArgs
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentDccTicketingValidationResultBinding
+import de.rki.coronawarnapp.ui.view.onOffsetChange
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.lists.diffutil.update
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModelsAssisted
+import javax.inject.Inject
+
+class DccTicketingValidationResultFragment : Fragment(R.layout.fragment_dcc_ticketing_validation_result), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    @Inject lateinit var validationResultAdapter: ValidationResultAdapter
+
+    private val binding: FragmentDccTicketingValidationResultBinding by viewBinding()
+    private val args: DccTicketingValidationResultFragmentArgs by navArgs()
+    private val resultViewModel: DccTicketingValidationResultViewModel by cwaViewModelsAssisted(
+        factoryProducer = { viewModelFactory },
+        constructorCall = { factory, _ ->
+            factory as DccTicketingValidationResultViewModel.Factory
+            factory.create(transactionContext = args.transactionContext)
```

Comment:
Please use the sharedViewModel to get the latest TransactionContext see https://github.com/corona-warn-app/cwa-app-android/pull/4439#discussion_r757413627

Reply:
Thanks, yes, I'm aware of this. This is actually the reason why the PR is still in draft mode.

Reply:
updated